src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+
+[Term]
+id: MONDO:1010258
+name: glomerulonephritis, non-human animal
+def: "Glomerulonephritis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000413 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000413"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0002462 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog glomerulonephritis
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:1010259
+name: nephritis, non-human animal
+def: "Nephritis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000708 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000708"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0001166 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog nephritis
+
+[Term]
+id: MONDO:1010260
+name: nephrolithiasis, non-human animal
+def: "Nephrolithiasis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000709 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000709"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008171 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog nephrolithiasis
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:1010261
+name: nephrotic syndrome, non-human animal
+def: "Nephrotic syndrome that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000712 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000712"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0005377 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog nephrotic syndrome
+
+[Term]
+id: MONDO:1010262
+name: polycystic kidney disease, non-human animal
+def: "Polycystic kidney disease that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000807 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000807"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000807"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0020642 {source="OMIA:000807"} ! cross-species analog polycystic kidney disease
+
+[Term]
+id: MONDO:1010263
+name: primary hyperoxaluria type II (Oxalosis II), non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.